### DISCUSSION TITLE
请问自定义容器支持嵌套吗？

### DISCUSSION THREAD

**Discussion by whiteChen233** [vuepress-theme-hope/vuepress-theme-hope#2291] [Q&A] [answered]
昨天写了页面，使用了容器嵌套，大概是这个样子
```md
::: tip
xxx

::: echarts
xxx
:::

ABCDEFG
:::
```

原本期望内容`ABCDEFG`在`tip`容器中显示，但是实际效果是`tip`容器在`echarts`容器那里结束了，导致后续内容在`tip`容器之外显示了，目前的解决方案是   我把`ABCDEFG`移到了`echarts`容器的前面  然而变成了下面这样（注意左下角那六个点）：

![image](https://user-images.githubusercontent.com/33564488/195576506-1a9b9a64-7031-4792-bbb9-69192bc8d36f.png)

所以，请问目前是否有办法解决？

**Comment by Mister-Hope** [comment, marked as answer]
来学习一下文档是怎么嵌套的 https://raw.githubusercontent.com/vuepress-theme-hope/vuepress-theme-hope/main/docs/md-enhance/src/guide/align.md

或者说，我认为你应该可以了解一个 markdown 基础，那就是基于marker的容器可以通过增加 nesting 来嵌套，文档的示例一直在这么用，如 https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/align.html#%E6%BC%94%E7%A4%BA

**Comment by whiteChen233** [comment]
get！示例上面看到了。

感谢！:smile:

第一个站我咋打不开呢...

**Reply by Gaobang100** [reply]
> get！示例上面看到了。
> 
> 感谢！😄
> 
> 第一个站我咋打不开呢...

太棒了这也解决了我的问题！！
挂梯子能打开[手动狗头]
